### ISSUE TITLE
Make release process aware of web store review latency

### ISSUE THREAD

**Issue by melink14** [melink14/rikaikun#981]
Right now, the publish request is instaneious and Github creates a release and marks issues and PRs as released with link to the chrome store.

This works fine except the actual item is not live in the store for 2-24 hours after the publish request is sent off.

It'd be nice to have a flow where the statuses were updated only after they were ready on the store.

Idea:
- Create a web store publish branch which we can run semantic release on but exclude Github plugin so notifications don't go out.
- Run a periodic action which checks versions and if they are live triggers a flow which merges publish branch into release branch.
- Have semantic release run on the release branch with Github and trigger the proper responses. Would require custom versions of the web store plugin so it realizes it doesn't need to publish.

One issue is that edge and chrome have different average review times. Not sure if I should have separate releases and branches for each one so they can launch independently or another solution.

Here's an NPM for checking webstore version: https://github.com/simov/chrome-webstore
